SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
+
+$currentLocation = Get-Location
+$packagePath = "$currentLocation\BuildOutput\$config\$arch\"
+
+if(!$task.CompareTo("i"))
+{
+	write-host "installing packages"
+	add-appxpackage "$packagePath\Microsoft.WindowsAppRuntime.Framework\Microsoft.WindowsAppRuntime.Framework.msix"
+	add-appxpackage "$packagePath\DynamicDependency.DataStore.Msix\DynamicDependency.DataStore.msix"
+	add-appxpackage "$packagePath\DynamicDependencyLifetimeManager.Msix\DynamicDependencyLifetimeManager.msix"
+	add-appxpackage "$packagePath\PushNotificationsLongRunningTask.Msix\PushNotificationsLongRunningTask.msix"
+}
+elseif(!$task.CompareTo("u"))
+{
+	taskkill /IM PushNotificationsLongRunningTask.exe /F
+	taskkill /IM DynamicDependency.DataStore.exe /F
+	taskkill /IM DynamicDependencyLifetimeManager.exe /F
```

Comment:
Why?

This only lives if the `MddBootstrapInitialize()` caller is running in an AppContainer. Otherwise the DDLMShadow.exe is used. And that kills itself if `MddBootstrapShutdown()` is called or the process that called `MddBootstrapInitalize()` terminates (DDLMShadow watches a process handler for the init'ing caller).